components/player-view.js

```diff
@@ -26,9 +27,13 @@ module.exports = React.createClass({
       progress: currentTime / window.AppData.duration
     });
 
-    if (currentTime >= window.AppData.duration) {
-      window.AppData.playing = false;
-      window.AppData.playedCount++;
+    // TODO: the "-1" is a hack to force YouTube embeds to do what we want.
```

Comment:
Is this the #184 issue? If so, we may want to add the bug number to the TODO comment for future context.